src/components/Contacts/ContactDetails/ContactDetails.graphql

```diff
@@ -0,0 +1,59 @@
+fragment ContactDetails on Contact {
+  id
+  ...ContactRow
```

Comment:
What's your thought process here?

Reply:
I was creating this query to load the data needed for the contact details screen, at the time I didn't see that there were many tabs off of this screen, I was just going off of what I saw in Figma.  So basically this data is for the Contact Details header as well as the tab labeled "Contact Details".  I decided to keep all of it because I thought "Contact Details" would be the first tab.  Maybe it would be good to keep this simpler and only load data for the header for now.

I decided to expand the ContactRow here because pretty much all of the properties on the ContactRow are also needed for this query as well.